sklearn/mixture/gmm.py

```diff
@@ -697,11 +692,11 @@ def _covar_mstep_full(gmm, X, responsibilities, weighted_X_sum, norm,
 def _covar_mstep_tied(gmm, X, responsibilities, weighted_X_sum, norm,
                       min_covar):
     # Eq. 15 from K. Murphy, "Fitting a Conditional Linear Gaussian
+    # Distribution"
     n_features = X.shape[1]
     avg_X2 = np.dot(X.T, X)
     avg_means2 = np.dot(gmm.means_.T, weighted_X_sum)
-    return (avg_X2 - avg_means2 + min_covar * np.eye(n_features)) / X.shape[0]
-
+    return (avg_X2 - avg_means2) / X.shape[0] + min_covar * np.eye(n_features) 
```

Comment:
I had several cases where inplace operations actually took longer than the copy operations for some reason.... but avoiding the ``eye`` makes a lot of sense.